README.md

```diff
@@ -20,157 +20,7 @@ External Contributors can read [Getting Started with OpenAPI Specifications](htt
 
 ## Directory Structure
 
-The structure of the directory should strictly follow these rules:
-
-1. **Profile**: The profile folder contains the profiles' definition files. The profile definition targets for hybrid applications that could run on Azure Stack general availability versions and Azure Cloud.
```

Comment:
@rkmanda, @JeffreyRichter any idea what's up with the `profile` and `profiles` dir? They haven't been updated since 3 years.

Reply:
The new document doesn't mention this at all now.